src/main/java/com/sprints/services/SprintsServiceImpl.java

```diff
@@ -88,6 +88,11 @@ public String createSprint(SprintDomain sprintDomain) {
 			Sprint sprint = sprintsValidationsRepository.oneSprintBacklogValidation();
 			sprintsValidations.sprintValidateInBacklog(sprint);
 		}
+		
+		if(sprintFinal.getActive()==true && sprintFinal.getIs_backlog() == true) {

```

Comment:
I think that this validation should be at the beginning, because, what is the objective of validate first if it is active and backlog if you will reject it ?